src/evaluations/run_evaluation.py

```diff
@@ -64,6 +64,10 @@
     'The name of this evaluation run.')
 flags.DEFINE_integer(
     'num_runs', None, 'The number of runs per scenario.', lower_bound=1)
+flags.DEFINE_integer(
+    'workers', 0, 
```

Comment:
should the default be the max?  This could have rough consequences for the user if they inadvertently spin up all the processes in their machine.  1 might be too much, but perhaps 2: what do you think, or would you still prefer 0?

Reply:
Considering that when we do run these evaluations we would want them to run as fast as possible, I don't think it's unjustified to use the maximum amount of processes that we can spin up. I see the possibility of accidentally using many fewer cores than you need as more damaging than using many more cores than you need. The consequences of using many fewer cores could be days lost while using many more cores than needed means that other people using the machine might see some slowdown in their work. Either way, it's recommended that you just set it to what you need, but the default is that the evaluations will go as fast as possible.

Reply:
right but I think the core of my question is: is the common case in terms of the number of times this script will be called, someone tinkering around with the evaluation script trying to do a test run, or is the common case the full shebang where the user wants things to go as fast as possible.  my suspicion is the former but could definitely be convinced otherwise